datastax-enterprise-marketplace/opscenter-install-resources.json

```diff
@@ -33,12 +33,13 @@
         "type": "CustomScriptForLinux",
         "typeHandlerVersion": "1.3",
```

Comment:
handlerVersion needs to be updated to 1.4 for all these templates with protected settings commandToExecute